test/geocoders/baidu.py

```diff
@@ -27,6 +27,7 @@ class BaiduTestCase(GeocoderTestBase):
     def setUpClass(cls):
         cls.geocoder = Baidu(
             api_key=env['BAIDU_KEY'],
+            security_key=env['BAIDU_SEC_KEY'],
```

Comment:
I suggest to create a separate test class instead, which does use the SK, so we could test both setups. I think that having two simple calls (1 geocode and 1 reverse) for this new class would be enough to be sure that SK works.

Also, if I understood correctly, that should be a different `BAIDU_KEY` (for which the SK auth is enabled), shouldn't it? If so, it should have a different ENV name.

Reply:
Yes, you're right. Are BAIDU_KEY_REQUIRES_SK and BAIDU_SEC_KEY ok?

Reply:
Yup, that's fine.